docs/topics/async.txt

```diff
@@ -395,3 +395,26 @@ Use with exception reporting filters
     If using these adapters with sensitive variables, ensure to audit exception
     reporting, and consider implementing a :ref:`custom filter
     <custom-error-reports>` if necessary.
+
+Async view decorators
+=====================
```

Comment:
The previous commit added a docs section on line 76. Please combine with that, by adding the new functions to its list.